sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -302,38 +313,85 @@ export class ServiceBusMessage implements ReceivedMessage {
     viaPartitionKey?: string;
 }
 
+// @public (undocumented)
+export interface ServiceBusReceiverClient {
+    new (queueAuth: QueueAuth, receiveMode: "peekLock"): ClientTypeT<"peekLock", "queue", "nosessions">;
+    new (queueAuth: QueueAuth, receiveMode: "receiveAndDelete"): ClientTypeT<"receiveAndDelete", "queue", "nosessions">;
+    new (queueAuth: QueueAuth, receiveMode: "peekLock", session: Session): ClientTypeT<"peekLock", "queue", "sessions">;
+    new (queueAuths: QueueAuth, receiveMode: "receiveAndDelete", session: Session): ClientTypeT<"receiveAndDelete", "queue", "sessions">;
+    new (subscriptionAuth: SubscriptionAuth, receiveMode: "peekLock"): ClientTypeT<"peekLock", "subscription", "nosessions">;
+    new (subscriptionAuth: SubscriptionAuth, receiveMode: "receiveAndDelete"): ClientTypeT<"receiveAndDelete", "subscription", "nosessions">;
+    new (subscriptionAuth: SubscriptionAuth, receiveMode: "peekLock", session: Session): ClientTypeT<"peekLock", "subscription", "sessions">;
+    new (subscriptionAuth: SubscriptionAuth, receiveMode: "receiveAndDelete", session: Session): ClientTypeT<"receiveAndDelete", "subscription", "sessions">;
+}
+
+// @public
+export const ServiceBusReceiverClient: ServiceBusReceiverClient;
+
+// @public (undocumented)
+export class ServiceBusSenderClient {
+    constructor(entityConnectionString: string, options?: ServiceBusClientOptions);
+    constructor(serviceBusConnectionString: string, entityName: string, options?: ServiceBusClientOptions);
+    constructor(host: string, entityName: string, credential: TokenCredential, options?: ServiceBusClientOptions);
+    // (undocumented)
+    cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
```

Comment:
I think we'd be okay to do that in EventHubs. I was trying to think if there are any issues but I think we're okay since we're just serializing the "number" as a string today in the checkpoint store.

Reply:
Filed #7616 to track it.